tests/acceptance/config/behat.yml

```
@@ -424,6 +424,332 @@ default:
         - TagContext:
         - TrashbinContext:
 
+    coreApiMain:
+      paths:
+        - "%paths.base%/../features/coreApiMain"
+      context: &common_ldap_suite_context
+        parameters:
+          ldapAdminPassword: admin
+          ldapUsersOU: TestUsers
+          ldapGroupsOU: TestGroups
+          ldapInitialUserFilePath: /../../config/ldap-users.ldif
+      contexts:
+        - FeatureContext: &common_feature_context_params
+            baseUrl: http://localhost:8080
+            adminUsername: admin
+            adminPassword: admin
+            regularUserPassword: 123456
+            ocPath: apps/testing/api/v1/occ
```

```suggestion
        - FeatureContext: *common_feature_context_params
```

The FeatureContext params are already like this at line 19. I think that we don't need to repeat them here.